src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,228 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
+import static core.basesyntax.MyHashMap.Node.hash;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static int CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        table = new Node[CAPACITY];
+        threshold = (int) (CAPACITY * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size > threshold) {
+            // resize();
+            //return;
+        }
+        int index = hash(key);
```

Comment:
The method `hash` should not be static and public. It's a utility method that is only used within the class, so it should be private. Also, make sure that you are using the diamond operator correctly when creating new instances of the Node class.